docs/openapi/components/schemas/common/AgricultureProduct.yml

```diff
@@ -120,6 +120,27 @@ properties:
     $linkedData:
       term: productImageHash
       '@id': https://w3id.org/traceability#productImageHash
+  variety:
+    title: Variety
+    description: "Additional details regarding the product. For example, in this case of potatoes this might include: russet, white, yellow flesh, etc."
+    type: string
+    $linkedData:
+      term: variety
+      '@id': https://www.gs1.org/voc/additionalProductDescription
+  commodityDesignation:
+    title: Commodity Designation
+    description: "Additional product subcategories such as: frozen, fresh, sliced, canned, dried, dehydrated, etc."
+    type: string
+    $linkedData:
+      term: commodityDesignation
+      '@id': https://www.gs1.org/voc/additionalProductDescription
+  packType:
+    title: Pack Type
+    description: "Details like: volume filled, tray pack, 18/lb consumer units, 10/5 pound bags, etc."
+    type: string
+    $linkedData:
+      term: packType
+      '@id': AAAA
```

Comment:
this is invalid

Reply:
Shoot, left this in by mistake... thanks for catching it